dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -771,6 +771,14 @@ ERROR_UNKNOWN_OSLIST_VALUE=CWWKF1387E: The value {0} is not a valid entry for th
 ERROR_UNKNOWN_OSLIST_VALUE.explanation=This value will be ignored as it is not a recognized operating system name.
 ERROR_UNKNOWN_OSLIST_VALUE.useraction=Please modify or remove this value from the feature manifest file.
 
+ERROR_ESA_NOT_FOUND=CWWKF1388E: The specified ESA file did not contain any features.
```

Comment:
It would be helpful for the message to have a parameter to say what file it is referring to.
e.g. `The specified ESA file {0} did not contain any features.` where {0} is the file path